### PR TITLE
Mock Tests for InvalidLinkBear 

### LINKED ISSUE #133: Mock tests for InvalidLinkBear
So right now the tests for InvalidLinkBear connects to the internet, instead of this  we should mock the requests, in order to make the tests faster and avoid failures in case of an unstable internet connection

### PR DESCRIPTION
**Pull request by abhsag24**
Fixes #133

### CODE REVIEW

**Comment by sils1297** [comment]
conflicts -> please rebase

**Comment by sils1297** [comment]
ack 56d515e ec6c00e

File: tests/general/InvalidLinkBearTest.py [hunk 1]
```diff
@@ -1,53 +1,113 @@
...
+        valid_file = """
+        http://httpbin.org/status/200
+        """.splitlines()
+
+        invalid_file = """
+        http://httpbin.org/status/301
+        http://coala.rtfd.org/302
+        """.splitlines()
+
+        self.assertResult(valid_file=valid_file, invalid_file=invalid_file)
```
Comment (sils1297) [inline tests/general/InvalidLinkBearTest.py]: for future reference you usually want to split this up into several test_ functions

**Comment by sils1297** [comment]
@rultor merge

File: tests/general/InvalidLinkBearTest.py (hunk 1 above)
Comment (abhsag24) [inline tests/general/InvalidLinkBearTest.py]: ahh thanks :)

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**